indico/htdocs/js/indico/jquery/categorynavigator.js

```diff
@@ -28,8 +28,19 @@
             openInDialog: false,
             // The title for the category navigator dialog
             dialogTitle: $T.gettext("Select a category"),
-            // Restrict action to categories with no subcategories
-            selectLeafOnly: false,
+            // Allow action to specific categories
+            enableActionOn: {
+                categoriesWithSubcategories: true,
+                categoriesWithEvents: true,
+                self: false
```

Comment:
`self` is the category for which the operation is performed. For example, when you move a category, self is the category that is being moved.